corehq/apps/userreports/models.py

```diff
@@ -83,6 +83,25 @@ def _check_ids(value):
         raise BadValueError("Invalid ID: '{}'".format(value))
 
 
+class UserReportActionLog(models.Model):
+    BUILD = 0
+    MIGRATE = 1
+    REBUILD = 2
+    DROP = 3
+
+    id = models.BigAutoField(primary_key=True)
+    domain = models.CharField(max_length=126, null=False, db_index=True)
+    indicator_config_id = models.CharField(max_length=126, null=False, db_index=True)
+    date_created = models.DateTimeField(auto_now_add=True, db_index=True)
+    action = models.PositiveSmallIntegerField(default=0, choices=(
```

Comment:
any reason for this to be an integer instead of a char field? it makes the queries a bit more comprehensible without needing extra constants (though would still be good to use constants with strings for creating actions). I don't believe that this table would be large enough to need the space savings

Reply:
yea, I guess since it's not going to be a huge table